### ISSUE TITLE
Disable grip as grip button in echo arena

### ISSUE THREAD

**Issue by jamesw6811** [LibreVR/Revive#730]
When using the excellent "TriggerAsGrip: true" functionality of the latest Revive, I've noticed that throwing is very difficult because not only does the trigger have to be released, but the grip has to be released also. When throwing, my hand naturally grips the controller harder, resulting in sticking. 

I'm new to the Revive project, but I'm happy to contribute a PR for this if someone can point me in the right direction.

**Comment by jLLOSAS** [comment]
Yeah is there a way to just switch the trigger and grip mechanisms? I like TriggerAsGrip but shouldn't there also be a GripAsTrigger options as well? This is strictly for Echo Arena. I can finally use the trigger to grab and throw things with TriggerAsGrip but then I can't punch or protect myself from being punched.

**Comment by jLLOSAS** [comment]
Ok NVM. So I have Grip on Normal Mode and TriggerAsGrip on True and you can punch with Trigger+Grip now and protect yourself. Only issue now is that at the beginning of a match it'll end up working but towards the end it ends up not working.

**Comment by gormanj** [comment]
The sticking issue seems to happen to me without using the trigger as grip function. I wonder what I am missing...

**Comment by wtesler** [comment]
Is there a way to punch using JUST grip? I find holding trigger and grip together is difficult.
